frontends/mit-learn/src/pages/ChannelPage/ChannelSearchFacetDisplay.tsx

```diff
@@ -117,7 +116,7 @@ const AvailableFacetsDropdowns: React.FC<
 
         return (
           facetItems.length && (
-            <StyledSelect
+            <SimpleSelect
```

Comment:
This code is not used anymore. The channel search just uses the regular search facets.